src/Cake.NuGet/NuGetContentResolver.cs

```diff
@@ -72,6 +76,11 @@ private IReadOnlyCollection<IFile> GetAddinAssemblies(DirectoryPath path, Packag
                 // Get relative path.
                 var relative = path.GetRelativePath(assembly.Path);
                 var framework = ParseFromDirectoryPath(current, relative.GetDirectory());
+                if (framework == null)
+                {
+                    _log.Warning("Unable to parse compatible framework from path {0}. Setting any framework.", assembly.Path.MakeAbsolute(_environment).FullPath);
```

Comment:
@mholo65 only slight comment would be the last part `Setting any framework`.  I know what you are trying to say, but not sure if that works.  Perhaps something like `, as a result, no framework will be specified.`